application-task-ui/src/main/resources/TaskManager/TaskManagerTranslations.xml

```diff
@@ -157,6 +157,8 @@ taskmanager.livetable.noRights=You don't have view rights for the document that
 
 taskmanager.macro.tasks.noRights=Some tasks have no been displayed because you don't have the rights to view them.
 
+taskmanager.picker.taskpage.noValue=Choose a reference to a task.
```

Comment:
The point of using a picker is to avoid the need to type / write the task page reference. In other words to hide the task page reference from the end user. The end user should simply select a task page.

```suggestion
taskmanager.picker.taskpage.noValue=Choose a task.
```